src/node_port.py

```diff
@@ -154,4 +155,61 @@ def delete_unit_service(self) -> None:
             else:
                 raise
 
+    @property
```

Comment:
functions brought over and modified from MySQL Router, PGBouncer, Kafka. (they were all slightly different so I tried to converge them + make them work for mine)